### PR TITLE
chore(build): clean out dist and add sourcemaps

### PR DESCRIPTION
**Pull request by jnewman**
Moving the test-reports out of dist as they're not part of the distribution. Also, adding sourcemaps
for .max.js.

### CODE REVIEW

File: test/karma-base.config.babel.js [hunk 1]
```diff
@@ -59,7 +59,9 @@ export default {
 
   singleRun: true,
 
-  webpack: webpackConfig,
+  webpack: Object.assign(webpackConfig, {
+    devtool: 'inline-source-map'
```
Comment (jnewman) [inline test/karma-base.config.babel.js]: I'm coming back around on sourcemaps. I'll add a task to debug w/o them in the future, but it's pretty annoying to debug webpack w/o them due to the noisy output.

**Comment by jnewman** [comment]
Unblocks https://github.com/cdnjs/cdnjs/pull/7709

File: test/karma-coverage.config.babel.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ module.exports = config => {
       }
     },
 
-    dir: 'dist/test-reports/coverage/',
+    dir: 'build/test-reports/coverage/',
```
Comment (sethyates) [inline test/karma-coverage.config.babel.js]: This should be ../build right?

**Comment by sethyates** [comment]
:+1: only 1 nit

File: test/karma-coverage.config.babel.js [hunk 3]
```diff
@@ -30,8 +30,6 @@ module.exports = config => {
       }
     },
 
-    dir: 'dist/test-reports/coverage/',
```
Comment (sethyates) [inline test/karma-coverage.config.babel.js]: Why removed?

Comment (jnewman) [inline test/karma-coverage.config.babel.js]: We're not actually using it here (none of our reporters actually write anything)

**Comment by sethyates** [comment]
LGTM

**Pull request merged**